x-pack/plugins/security_solution/public/users/pages/navigation/user_risk_score_tab_body.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { noop } from 'lodash/fp';
+
+import { UsersComponentsQueryProps } from './types';
+import { manageQuery } from '../../../common/components/page/manage_query';
+import { useDeepEqualSelector } from '../../../common/hooks/use_selector';
+import { State } from '../../../common/store';
+import { HostRiskScoreQueryId } from '../../../common/containers/hosts_risk/types';
+
+import { useUserRiskScore } from '../../../hosts/containers/host_risk_score';
+import { useRiskScoreKpi } from '../../../hosts/containers/kpi_hosts/risky_hosts';
+import { UserRiskScoreTable } from '../../components/user_risk_score_table';
+import { usersSelectors } from '../../store';
+
+const UserRiskScoreTableManage = manageQuery(UserRiskScoreTable);
+
+export const UserRiskScoreQueryTabBody = ({
+  endDate,
+  filterQuery,
+  skip,
+  setQuery,
+  startDate,
+  type,
+  deleteQuery,
+}: UsersComponentsQueryProps) => {
+  const getUserRiskScoreSelector = useMemo(() => usersSelectors.userRiskScoreSelector(), []);
+  const { activePage, limit, sort } = useDeepEqualSelector((state: State) =>
+    getUserRiskScoreSelector(state)
+  );
+
+  const pagination = useMemo(
+    () => ({
+      cursorStart: activePage * limit,
+      querySize: limit,
+    }),
+    [activePage, limit]
+  );
+
+  const [loading, { data, totalCount, inspect, isInspected, refetch }] = useUserRiskScore({
+    filterQuery,
+    skip,
+    pagination,
+    sort,
+  });
+
+  const { severityCount, loading: isKpiLoading } = useRiskScoreKpi({
```

Comment:
this will need to take a type argument as well, user or host

Reply:
It was outdated. Now I am calling `useUserRiskScoreKpi`.